client/src/Auth/components/Login.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const Login = ({ handleClickOther }) => {
+  const [userText, setUserText] = useState('');
+  const [passText, setPassText] = useState('');
+
+  const handleUserChange = (e) => {
+    setUserText(e.target.value);
+  };
+
+  const handlePassChange = (e) => {
+    setPassText(e.target.value);
+  };
+
+  return (
+    <>
+      <h1>Log in to Greene Ocean</h1>
+      <h3>Username</h3>
+      <input
+          id="search"
+          type="search"
+          autoComplete="off"
+          maxLength="150"
+          style={{
+            height: '40px',
+            width: '500px',
+            borderRadius: '6px',
+            padding: '12px',
+            backgroundColor: 'rgb(245,245,245)',
+            transition: 'all 0.5s',
+            cursor: 'text',
+          }}
+          value={userText}
```

Comment:
@cgf5033 instead of styling the input inline maybe offloading it to a styled component (styled.input`...`) so it can be reused and is a but cleaner.